authentication/account/repository/identity.go

```diff
@@ -156,11 +156,11 @@ func (m Membership) TableName() string {
 
 // Load returns a single Identity as a Database Model
 // This is more for use internally, and probably not what you want in  your controllers
-func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID) (*Identity, error) {
+func (m *GormIdentityRepository) Load(ctx context.Context, id uuid.UUID, funcs ...func(*gorm.DB) *gorm.DB) (*Identity, error) {
```

Comment:
Add a param description saying that funcs can be used to add dynamic conditions to all methods where you are adding this param.

Reply:
Addressed in 72b2576